constellations/gemini-admin/src/programs/programRow.styl

```diff
@@ -1,47 +1,31 @@
 @import "../static.styl"
 
-#view-program #list-heading
-  border-bottom: 2px solid colorGrayDark
-
-  & li
+#view-program
+  & span
     display: inline-block
-    height: spacing4x
-    line-height: spacing4x
 
-  & button
-    color: colorTurquoise
-    
-#view-program
-  & .li-small
+  .small
     width: 80px
-  & .li-med
+
+  .medium
     width: 160px
-  & .li-large
-    width: 240px
-  & .li-checkbox
-    width: 100px
 
-#view-program #list-rows li
-  display: block
-  position: relative
-  width: 100%
-  height: 60px
-  line-height: 60px
-  border-bottom: 1px solid colorGrayLight
+  .large
+    width: 240px
 
-  &.unpublished
-    color: colorGrayMed
+  #header
+    display: flex
+    height: 55px
+    align-items: center
+    border-bottom: 2px solid colorGrayDark
```

Comment:
Take a look here about CSS ordering. It helps organize properties and where to quickly look for them 
https://github.com/ahsu1230/mathnavigatorSite/blob/master/resources/frontend/css_ordering.md

Reply:
Where does `align-items` go in that list?